app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less

```diff
@@ -122,9 +122,10 @@
                 &:before {
                     content: @icon-expand-close__content;
                 }
+
                 &.active {
                     &:before {
-                        content: @icon-expand-open__content;
```

Comment:
actually this additional is unnecessary. With class _show or opened already enough can cover this. We don't need more specific with active here

Reply:
Hi @mrtuvn : Thank you. I think you can remove the .active css line.